PaddleCV/object_detection/README.md

```diff
@@ -1,4 +1,130 @@
+
+- High Efficiency:
+  Based on high efficient PaddlePaddle framework, less memory is required. For example, the batch size of Mask-RCNN based on ResNet50 can be 5 per Tesla V100 (16G). The training speed of Yolo v3 is faster than other framework.  
+
+
+
+The supported architectures are as follows:
+
+
+|                    | ResNet |ResNet vd| ResNeXt  | SENet    | MobileNet | DarkNet|
+|--------------------|:------:|--------:|:--------:|:--------:|:---------:|:------:|
+| Faster R-CNN       | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
+| Faster R-CNN + FPN | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
+| Mask R-CNN         | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
+| Mask R-CNN + FPN   | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
+| Cascade R-CNN      | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
+| RetinaNet          | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
+| Yolov3             | ✓      | ✗      | ✗         |  ✗       | ✓        | ✓     |
+| SSD                | ✗      | ✗      | ✗         |  ✗       | ✓        | ✗      |
+
+The extensive capabilities:
+
+- [x] **Synchronized batch norm**:  used in Yolo v3.
+- [x] **Group Norm**: supported this operation, the related model will be added later.
+- [x] **Dodulated deformable convolution**: suppored this operation, the related model will be added later.
+- [x] **Deformable PSRoI Pooling**: suppored this operation, the related model will be added later.
+
+
+#### Work in Progress and to Do
+
+- About Framework:
+   - Mixed precision training and distributed training.
+   - 8-bit deployment.
+   - Easy to customize user-defined function.
+
+- About Algorithms:
+   - More SOTA models.
+   - More easy-to-deployed models.
+
+
+We are also pleased to receive your feedback.
+
+## Model zoo
+
+The trained models can be available in PaddlePaddle [detection model zoo](docs/MODEL_ZOO.md).
+
+## Installation
+
+Please follow the [installation instructions](docs/INSTALL.md) to install PaddlePaddle and prepare environment.
+
+## Get Started
+
+For quickly start, infer a image:
+
+```bash
+export PYTHONPATH=`pwd`:$PYTHONPATH
+python tools/infer.py -c configs/mask_rcnn_r50_1x.yml \
+    -o weights=http://
+    -
+```
+
+The predicted result is visualized in `output/xxx.jpg`.
+
+For more detailed training and evaluating pipeline, please refer [GETTING_STARTED.md](docs/GETTING_STARTED.md).
+
+
+For more documentation, please refer:
+
+- [How to config an object detection pipeline.](docs/CONFIG.md)
+- [How to use customized dataset and add data preprocessing.](docs/DATA.md)
+
+
+## Deploy
+
+The example how to use PaddlePaddle to deploy detection model will be added later.
+
+
+## Framework
+
+The code tree is as follows.
```

Comment:
IMHO, this is should not be included for 2 reasons
- folder name are quite self explanatory 
- the directory structure is still subject to future change